security.md

```diff
@@ -12,11 +12,23 @@ Security announcements related to Scala are published to the ["Security Announce
 Messages to this channel can only be posted by administrators, so it is very low traffic.
 To set up email notifications for new security announcements, read [this post](https://users.scala-lang.org/t/about-the-security-announcements-category).
 
+## Releases Integrity
+
+To ensure the integrity of all the releases, our organization uses [PGP](https://gnupg.org/) keys for cryptographic signing. 
+We provide below an **exhaustive** list of all the keys
+used for signing the artifacts under the `org.scala-lang` namespace on [Maven Central](https://central.sonatype.com/namespace/org.scala-lang)
+
+|                    Fingerprint                    | Algorithm |                                                       Public Key                                                        |
+|:-------------------------------------------------:|:---------:|:-----------------------------------------------------------------------------------------------------------------------:|
+| 86DA 41A5 E169 9C9C EBE9 64A8 A905 2B1B 6D92 E560 | RSA-4096  | [Download Public Key](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x86da41a5e1699c9cebe964a8a9052b1b6d92e560) |
+| ACF3 9CCD ED38 E2C6 F089 8BF2 8F7F 6C04 5196 7B84 | RSA-4096  | [Download Public Key](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xacf39ccded38e2c6f0898bf28f7f6c0451967b84) |
+| 2A5E 8B33 8438 CAC7 033F 9D8F B8A0 45C0 A6EC 398E | RSA-4096  | [Download Public Key](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2a5e8b338438cac7033f9d8fb8a045c0a6ec398e) |
+
 ## Reporting Vulnerabilities
 
 We strongly encourage reporting security issues in Scala to us privately before disclosing them in public.
 
-The email address for security related communication is `security@scala-lang.org`.
+The email address for security related communication is [`security@scala-lang.org`](mailto:security@scala-lang.org).
```

Comment:
I've sent message _Please publish updated PGP key used to sign scala-library in central_ to this address on May 10th, but got:
```
We're writing to let you know that the group you tried to contact (scala-security) may not exist, or you may not have permission to post messages to the group. A few more details on why you weren't able to post:

 * You might have spelled or formatted the group name incorrectly.
 * The owner of the group may have removed this group.
 * You may need to join the group before receiving permission to post.
 * This group may not be open to posting.

If you have questions related to this or any other Google Group, visit the Help Center at https://groups.google.com/support/.

Thanks,

Google Groups
```

Reply:
as best we can determine, whatever happened there, back then, has been fixed since